iml-services/iml-ntp/cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "iml-ntp"
+version = "0.1.0"
+authors = ["IML Team <iml@whamcloud.com>"]
+edition = "2018"
+
+[dependencies]
+lapin-futures = "0.18.0"
+tokio = "0.2.0-alpha.4"
+tokio-timer = "0.2.11"
+tokio-postgres = "0.4.0-rc.3"
+env_logger = "0.6.1"
+log = "0.4.6"
```

Comment:
use tracing, tracing-subscriber instead.

Reply:
Remove both env_logger and log